### ISSUE TITLE
Show a warning if a <section> does not have a heading element within it

### ISSUE THREAD

**Issue by jackdomleo7** [jackdomleo7/Checka11y.css#89]
### Describe the new a11y feature or project enhancement

According to MDN and w3, a `<section>` "should" contain a heading element (the heading element is used to describe the `section`). This should only be a warning and not an error because MDN and w3 both state "should" and not "must"; it's an advisory rather than a requirement.

### Link(s)

- https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section

**Comment by shamahdotdev** [comment]
Hey there, I can help you with this feature, can you assign it to me?

**Comment by jackdomleo7** [comment]
@shamahdotdev certainly. I've had a couple of attempts at this already and couldn't figure it out so would be interesting if you can figure it out 🙂

**Comment by shamahdotdev** [comment]
Thank you @jackdomleo7, and yes it would've been easier to use `:has()` pseudo selector but unfortunately it's still on experiment and no browser was supported yet. So my attempt is to paint all last child element on a section with warning messages and remove it if heading element exist, i also use `:before` to make sure it didn't conflict with other error/warning. I hope it's find because i think there's no better implementation for it.

**Comment by jackdomleo7** [comment]
I like the idea of `:has()`, shame it has 0% support on caniuse.com. But I think your suggested approach is a good one for now. I'd be happy to look at your PR now.
